src/AceEditor/madie-ace-editor.tsx

```diff
@@ -289,9 +289,7 @@ const MadieAceEditor = ({
     if (isParsing) {
       return <span>Parsing...</span>;
     } else if (editorAnnotations && editorAnnotations.length > 0) {
-      return (
-        <span>{editorAnnotations.length} issues were found with CQL...</span>
-      );
+      return "";
```

Comment:
Based on my understanding,  we don't want to show any message footer below the editor. 
1. If this the case we can remove the entire renderFooterMsg function 
2. or else if we decide we want to show the parsing message in footer we can remove the entire else if as we don't need that condition now(as if we are not showing any issues found info)